hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BinaryBackendEntry.java

```diff
@@ -42,7 +42,14 @@
     private final List<BackendColumn> columns;
 
     public BinaryBackendEntry(HugeType type, byte[] bytes) {
-        this(type, BytesBuffer.wrap(bytes).parseId());
+        if (type.isIndex()) {
+            id = BytesBuffer.wrap(bytes).readIndexId();
```

Comment:
change to:
` type.isIndex() ？BytesBuffer.wrap(bytes).readIndexId() : BytesBuffer.wrap(bytes).parseId();`